mmv1/products/iambeta/WorkloadIdentityPoolNamespace.yaml

```diff
@@ -0,0 +1,135 @@
+  - name: 'iam_workload_identity_pool_namespace_basic'
+    primary_resource_id: 'example'
+    vars:
+      workload_identity_pool_id: 'example-pool'
+      workload_identity_pool_namespace_id: 'example-nmspc'
+  - name: 'iam_workload_identity_pool_namespace_full'
+    primary_resource_id: 'example'
+    vars:
+      workload_identity_pool_id: 'example-pool'
+      workload_identity_pool_namespace_id: 'example-nmspc'
+parameters:
+  - name: 'workload_identity_pool_id'
+    type: String
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The ID to use for the pool, which becomes the final component of the resource name. This
+      value should be 4-32 characters, and may contain the characters [a-z0-9-]. The prefix
+      `gcp-` is reserved for use by Google, and may not be specified.
+  - name: 'workload_identity_pool_namespace_id'
+    type: String
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The ID to use for the namespace. This value must:
+      * contain at most 63 characters
+      * contain only lowercase alphanumeric characters or `-`
+      * start with an alphanumeric character
+      * end with an alphanumeric character
+
+
+      The prefix `gcp-` will be reserved for future uses.
+    validation:
+      function: 'ValidateWorkloadIdentityPoolNamespaceId'
+properties:
+  - name: 'name'
+    type: String
+    description: |
+      The resource name of the namespace as
+      `projects/{project_number}/locations/global/workloadIdentityPools/{workload_identity_pool_id}/namespaces/{workload_identity_pool_namespace_id}`.
+    output: true
+  - name: 'description'
+    type: String
+    description: |
+      A description of the namespace. Cannot exceed 256 characters.
+    validation:
+      function: 'ValidateWorkloadIdentityPoolNamespaceDescription'
+  - name: 'state'
+    type: Enum
+    description: |
+      The current state of the namespace.
+      * `STATE_UNSPECIFIED`: State unspecified.
+      * `ACTIVE`: The namespace is active.
+      * `DELETED`: The namespace is soft-deleted. Soft-deleted namespaces are permanently deleted
+      after approximately 30 days. You can restore a soft-deleted namespace using
+      UndeleteWorkloadIdentityPoolNamespace. You cannot reuse the ID of a soft-deleted namespace
+      until it is permanently deleted.
+    output: true
+    enum_values:
+      - 'STATE_UNSPECIFIED'
+      - 'ACTIVE'
+      - 'DELETED'
+  - name: 'disabled'
+    type: Boolean
+    description: |
+      Whether the namespace is disabled. If disabled, credentials may no longer be issued for
+      identities within this namespace, however existing credentials will still be accepted until
+      they expire.
+  - name: 'ownerService'
+    type: NestedObject
+    description: |
+      Defines the owner that is allowed to mutate this resource. If present, this resource can only
+      be mutated by the owner.
+    output: true
```

Comment:
Is the field `ownerService` output only? If yes, the nested field `principalSubject` should also have `output: true`. Or `principalSubject` can be provided by the users?